.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "libbpf"]
 	path = tracee-ebpf/3rdparty/libbpf
 	url = https://github.com/libbpf/libbpf
+[submodule "btfhub"]
```

Comment:
> I'm not a fan of git submodules for build-phase dependencies. most if not all people who choose to build tracee won't build btfs, and considering the size of btfhub cloning it is not fun.

I have thought about this also. We could have btfhub cloned through the makefile option (instead of submodule). This way, a git clone command including recursive submodules would not clone btfhub.